examples/CMakeLists.txt

```diff
@@ -123,12 +123,21 @@ if (NOT USING_OLD_VISUAL_STUDIO_COMPILER)
    add_gui_example(dnn_imagenet_ex)
    add_gui_example(dnn_mmod_find_cars_ex)
    add_example(dnn_mmod_train_find_cars_ex)
+   add_gui_example(dnn_semantic_segmentation_ex)
+   add_example(dnn_semantic_segmentation_train_ex)
```

Comment:
Note, before merging:
- Remove this line
- Uncomment the corresponding line below